src/meta/processors/admin/HBProcessor.cpp

```diff
@@ -34,7 +36,7 @@ void HBProcessor::process(const cpp2::HBReq& req) {
             << ", role = " << apache::thrift::util::enumNameSafe(role);
 
   if (role == cpp2::HostRole::STORAGE) {
-    if (!ActiveHostsMan::machineRegisted(kvstore_, host)) {
+    if (!FLAGS_auto_registe && !ActiveHostsMan::machineRegisted(kvstore_, host)) {
```

Comment:
I have a problem
For other roles such as storage_listener, do I need to add hosts?

Reply:
In my test, other roles are not cpp2::HostRole::STORAGE, add host succeeds, but show does not come out

Reply:
https://github.com/vesoft-inc/nebula/pull/3444  have fix it

Reply:
I think you should merge that bug in first, otherwise, the entire cluster will be useless.